autogpt_platform/backend/backend/usecases/block_autogen.py

```diff
@@ -124,7 +124,7 @@ def create_test_graph() -> Graph:
 
 Here are a couple of sample of the Block class implementation:
 
-{"--------------\n".join([sample_block_codes[v] for v in sample_block_modules])}
+{"--------------".join([sample_block_codes[v] for v in sample_block_modules])}
```

Comment:
what's this?

Reply:
It causes the current pre-commits to fail due to having a \ in an f-string